### PR TITLE
Add more examples of operation specification language

### PR DESCRIPTION
**Pull request by bkamins**
Fixes https://github.com/JuliaData/DataFrames.jl/issues/3086

### CODE REVIEW

**Comment by JakeZw** [comment]
Thank you.  As a new user I was looking for a way to use an equation on values from more than one column.  Apparently there are other ways than what I came up with, so showing these will be helpful to new users. 

This is the first time I really used DataFrames, and it seems to have much potential.  Thank you for your continued development of this tool.

File: docs/src/man/working_with_dataframes.md [hunk 1]
```diff
@@ -553,6 +553,22 @@ julia> select(df, :x2, :x2 => ByRow(sqrt)) # transform columns by row
    1 │     3  1.73205
    2 │     4  2.0
 
+julia> select(df, :x1, :x2, [:x1, :x2] => ((x1, x2) -> x1 ./ x2) => :z) # transformation of multiple columns
```
Comment (nalimilan) [inline docs/src/man/working_with_dataframes.md:556]: For consistency with above:
```suggestion
julia> select(df, :x1, :x2, [:x1, :x2] => ((x1, x2) -> x1 ./ x2) => :z) # transform multiple columns
```

File: docs/src/man/working_with_dataframes.md [hunk 2]
```diff
@@ -553,6 +553,22 @@ julia> select(df, :x2, :x2 => ByRow(sqrt)) # transform columns by row
...
+2×3 DataFrame
+ Row │ x1     x2     z
+     │ Int64  Int64  Float64
+─────┼────────────────────────
+   1 │     1      3  0.333333
+   2 │     2      4  0.5
+
+julia> select(df, :x1, :x2, [:x1, :x2] => ByRow((x1, x2) -> x1 / x2) => :z)  # transformation of multiple columns by row
```
Comment (nalimilan) [inline docs/src/man/working_with_dataframes.md:564]:
```suggestion
julia> select(df, :x1, :x2, [:x1, :x2] => ByRow((x1, x2) -> x1 / x2) => :z)  # transform multiple columns by row
```

**Review by nalimilan — APPROVED** [review]

**Comment by bkamins** [comment]
Thank you!

**Pull request merged**